sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/VirtualMachinesOperationsExtensions.cs

```diff
@@ -1187,9 +1193,12 @@ public static async Task<VirtualMachine> BeginUpdateAsync(this IVirtualMachinesO
             /// <param name='vmName'>
             /// The name of the virtual machine.
             /// </param>
-            public static void BeginDelete(this IVirtualMachinesOperations operations, string resourceGroupName, string vmName)
+            /// <param name='forceDeletion'>
+            /// Optional parameter to force delete virtual machines.(Feature in Preview)
+            /// </param>
+            public static void BeginDelete(this IVirtualMachinesOperations operations, string resourceGroupName, string vmName, bool? forceDeletion = default(bool?))
```

Comment:
Same comment

Reply:
please confirm if generated files can be edited.